### DISCUSSION TITLE
Webserver on Windows?

### DISCUSSION THREAD

**Discussion by lufusol** [jef/streetmerchant#2490] [Help]
I'm unclear on how to start the local web daemon feature.  I configured WEB_PORT= in dotenv with an integer, a port number.  Is that correct or is this just a true/false value and if so what port ought it to be running on, default (80)?  

Is there anything I need to drop into the streetmerchant dir to enable the web daemon to run under Windows?

**Comment by Aragorn450** [comment]
It should just work after you start the server with `npm run start`. The web server doesn't start until the very end, just before it starts checking the stores though so wait until you see it returning results from the stores and then load the page. You should probably type in the http:// part into your browser so it knows you're not trying so search so.... type in http://localhost/ if you use port 80 or http://localhost:8080/ for example if you use 8080.

It's also possible that another service is using port 80 (I've seen Skype use it for instance), so you might want to try another port. 8080 and 8888 are common ones people use for alternate ports for a web server.

Hope that all helps out!
